js-intro.js

```diff
@@ -11,6 +11,34 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+// making the boiling point
+var boilPoint = 212
+
+// creating a function for each temp
+const isItBoiling1 = (isitboiling) => {
+    if(temp1 < boilPoint ) {
+        return `${temp1} is below the boiling point`
+    }
+}
+const isItBoiling2 = (isitboiling) => {
+    if(temp2 > boilPoint ) {
+        return `${temp2} is above the boiling point`
+    }
+}
+const isItBoiling3 = (isitboiling) => {
+    if(temp3 || boilPoint ) {
+        return `${temp3} is at the boiling point`
+    }
+}
+
```

Comment:
Rather than creating 3 different functions, how can you create one single function that can verify if _any_ integer I pass in is at, above, or below boiling point? HINT ** if/else decision tree ** 
The logic for your third function doesn't really work. The one checking if the number is at boiling point.
The argument you have here `isitboiling` isn't being utilized inside the function so I'm not sure what you wanted to do with that argument. If you need help with this let me know :)